canopeum_frontend/src/components/analytics/OptionQuantitySelector.scss

```diff
@@ -7,3 +7,16 @@
     top: 0px;
   }
 }
+
+input[type=number].no-spinner {
+  -webkit-appearance: none; // WebKit browsers
+  appearance: none; // Standard property
+  -moz-appearance: textfield; // Firefox
```

Comment:
Do we not have a bundler that takes care of setting these automatically just based on `appearance` ? Maybe I spent too much time in Angular ^^" Would be worth looking into (our entire CSS quality checking and automated tooling is generally lacking, Esteban has been looking into it recently)

Reply:
Not sure about this, could be worth checking/investigating. I think this can be done with postcss. However, It think this should be done in a separate issue as it require some configuration changes. Also, while doing that we could standardize all the css.

Reply:
Yeah not something to do here. This was just a comment/remark.